tests/integration/integration/home/home-en.spec.js

```diff
@@ -0,0 +1,23 @@
+import { Selector } from "testcafe";
+import { googleAuthUser } from "../../helpers/loginByGoogle";
+import { selectLocaleFromDialog } from "../../helpers/selectLocale";
+
+fixture("Home - English")
+  .page(`${process.env.BROWSERSTACK_BASE_URL}/login`)
+  .beforeEach(async (t) => {
+    await t
+      .useRole(googleAuthUser)
+      .navigateTo(`${process.env.BROWSERSTACK_BASE_URL}/home`);
```

Comment:
let's generally avoid using `t` everywhere as:
- not a meaningful variable name
- one could confuse it with the `t` from `i18n`

Reply:
Done.